modules/saml/lib/Auth/Source/SP.php

```diff
@@ -58,7 +66,8 @@ public function __construct($info, $config) {
 		$this->entityId = $this->metadata->getString('entityID');
 		$this->idp = $this->metadata->getString('idp', NULL);
 		$this->discoURL = $this->metadata->getString('discoURL', NULL);
-		
+		$this->disableScoping = $this->metadata->getBoolean('disable.scoping', NULL);
```

Comment:
Would it make sense to default to `FALSE` instead of `NULL`? Keeps the type of the var above also simpler.